src/makernote_int.cpp

```diff
@@ -1190,6 +1198,18 @@ namespace Exiv2 {
         };
         return std::find(std::begin(models), std::end(models), getExifModel(pRoot)) != std::end(models) ? 0 : -1;
     }
+    int sony2FpSelector(uint16_t /*tag*/, const byte* /*pData*/, uint32_t /*size*/, TiffComponent* const pRoot)
+    {
+        // Not valid for models beginning
+        const char* models[] = { "SLT-", "HV", "ILCA-" };
+
+        std::string model = getExifModel(pRoot);
+        for (auto&& m : models) {
```

Comment:
```suggestion
        for (auto& m : models) {
```

What do you think about just using `auto&` here ? 
I'm not a fan of using `auto&&` in range based for's unless one actually needs to bind to a rvalue.   
If that isn't necessary, I think it's easier to read and understand `auto&` because `auto&&` just makes people wonder if there is a special thing going on that they didn't yet figure out.

Reply:
Actually, you could even do: 
```cpp
for (auto& m : { "SLT-", "HV", "ILCA-" }) {
```

Reply:
I have made the change.